bims/admin.py

```diff
@@ -131,11 +136,13 @@ def response_add(self, request, obj, post_url_continue=None):
             [obj.email],
             fail_silently=False
         )
-        return redirect('/admin/auth/user/')
+        return super(CustomUserAdmin, self).response_add(
```

Comment:
Change redirect to call overridden class function.
We need to call parent's method.